maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/mapper/ProceedingsMapper.java

```diff
@@ -123,16 +138,19 @@ private ApiPassportAssessment applicationDtoToPassportAssessment(ApplicationDTO
         return null;
     }
 
-    private ApiFinancialAssessment applicationDtoToFinancialAssessment(ApplicationDTO application) {
+    private ApiFinancialAssessment applicationDtoToFinancialAssessment(ApplicationDTO application, boolean isMeansAssessment) {
 
         FinancialAssessmentDTO financialAssessmentDTO = application.getAssessmentDTO().getFinancialAssessmentDTO();
         FullAssessmentDTO fullAssessment = financialAssessmentDTO.getFull();
         InitialAssessmentDTO initialAssessment = financialAssessmentDTO.getInitial();
 
         ApiFinancialAssessment assessment = new ApiFinancialAssessment();
-        assessment
-                .withInitResult(initialAssessment.getResult())
-                .withInitStatus(CurrentStatus.getFrom(initialAssessment.getAssessmnentStatusDTO().getStatus()));
+        assessment.withInitResult(initialAssessment.getResult());
+        if (isMeansAssessment) {
+            assessment.withInitStatus(CurrentStatus.getFrom(initialAssessment.getAssessmnentStatusDTO().getStatus()));
+        } else {
+            assessment.withInitStatus(getCurrentStatus(initialAssessment.getAssessmnentStatusDTO().getStatus()));
```

Comment:
See previous comment, we should just be calling `getCurrentStatus`, no need for branching logic.